css/styles.css

```diff
@@ -0,0 +1,299 @@
+HTML{
+    background-color: rgba(45, 46, 46, 0.767);
+}
+header{
+    background-color: aqua;
+    position: absolute;
+    width: 1920px;
+    height: 840px;
+    left: 0px;
+    top: 0px;
+}
+.JK{
+    position: absolute;
+    width: 556px;
+    height: 109px;
+    left: 682px;
+    top: 296px;
+    
+    font-family: Playfair Display;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 64px;
+    text-align: center;
+    color: #D4C17F;
+    }
+header.inner{
+    width: 1080px;
+    margin: 0 auto;
+    position: relative;
+    height: 400px;
+}
+.nav{
+    position: absolute;
+    top: 20px;
+    right:120px;
+}
+.nav_list{
```

Comment:
Use [BEM notation](https://ru.bem.info/methodology/html/). In this case `.nav-list`.